novawallet/Modules/Staking/StartStakingInfo/Model/StartStakingViewModelFactory.swift

```diff
@@ -63,12 +127,40 @@ struct StartStakingViewModelFactory: StartStakingViewModelFactoryProtocol {
         )
     }
 
-    func rewardModel(locale: Locale) -> ParagraphView.Model {
-        let rewardIntervals = "every 6 hours"
-        let text = R.string.localizable.stakingStartRewards(
-            rewardIntervals,
-            preferredLanguages: locale.rLanguages
-        )
+    func rewardModel(
+        stakingType: StartStakingType,
+        chainAsset: ChainAsset,
+        eraDuration: TimeInterval,
+        locale: Locale
+    ) -> ParagraphView.Model {
+        let separator = R.string.localizable.commonAnd(preferredLanguages: locale.rLanguages)
+        let preposition = R.string.localizable.commonTimePeriodEvery(preferredLanguages: locale.rLanguages)
+        let time = eraDuration.localizedDaysHoursMinutes(for: locale, separator: separator)
+        let rewardIntervals = [
+            preposition,
+            time
+        ].joined(with: .space)
+        let text: String
+
+        switch stakingType {
```

Comment:
Why do we need to separate logic between direct staking and nomination pools? I see that we have the same text in the mockups.